src/riscv-legacy-integration.adoc

```diff
@@ -345,6 +370,12 @@ The CHERI Mode Enable (CME) bit controls whether less privileged levels (e.g.
 S-mode and U-mode) execute in Capability or Legacy mode. When CME=1, the
 CHERI execution mode is Capability. When CME=0, the mode is Legacy.
 
+The CHERI Register Enable (CRE) bit controls whether less privileged levels can
```

Comment:
I wonder if this would fit better into the state enable extension since conceptually this is about making the additional state accessible? E.g. as a bit in `mstateen`/`sstateen`?

Reply:
Potentially, but once there's consensus on the approach here the bit is just going to get deleted by phase 2 so I'd rather not focus on where it lives until then.

Reply:
mstateen/stateen does seem more sensible, but I agree with @sorear that anything we allocate will be moved later, so maybe it's not important where they go

Reply:
accepting that we don't currently have banked DDC let's go with this proposal for now